R/html-tweak.R

```diff
@@ -159,7 +159,10 @@ tweak_rmarkdown_html <- function(html, input_path, pkg = pkg) {
   invisible()
 }
 
-tweak_homepage_html <- function(html, strip_header = FALSE, sidebar = TRUE) {
+tweak_homepage_html <- function(
+  html, strip_header = FALSE, sidebar = TRUE,
+  bs_version
+  ) {
```

Comment:
If you're going to spread the arguments over multiple lines, I think you should probably put one argument on each line (and `bs_version` should get a default since it comes after another argument with a default)